docs/examples/test/test_examples.py

```diff
@@ -286,27 +286,26 @@ def test_websocket_rpc(self, container_factory, web_config, websocket):
 
 class TestConfig:
 
-    def test_config_value_not_set(self, container_factory, empty_config):
+    def test_config_value_not_set(self, container_factory, rabbit_config):
```

Comment:
hm. ok. (given how relative expensive it is to spin use the rabbit dep, maybe we can make those examples not need it)